packages/builders/src/interactions/commands/chatInput/ChatInputCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { ApplicationCommandType, type RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+import { Mixin } from 'ts-mixer';
+import { isValidationEnabled } from '../../../util/validation.js';
+import { CommandBuilder } from '../Command.js';
+import { SharedNameAndDescription } from '../SharedNameAndDescription.js';
+import { chatInputCommandPredicate } from './Assertions.js';
+import { SharedChatInputCommandOptions } from './mixins/SharedChatInputCommandOptions.js';
+import { SharedChatInputCommandSubcommands } from './mixins/SharedSubcommands.js';
+
+/**
+ * A builder that creates API-compatible JSON data for chat input commands.
+ */
+export class ChatInputCommandBuilder extends Mixin(
```

Comment:
I think this should also have an `options` getter

Reply:
resolved in a child class